source/ui/BioXAS/BioXASSideXASScanConfigurationView.cpp

```diff
@@ -40,8 +40,8 @@ BioXASSideXASScanConfigurationView::BioXASSideXASScanConfigurationView(BioXASSid
 {
 	configuration_ = configuration;
 
-	topFrame_ = new AMTopFrame("Configure an XAS Scan");
-	topFrame_->setIcon(QIcon(":/utilities-system-monitor.png"));
+	//topFrame_ = new AMTopFrame("Configure an XAS Scan");
```

Comment:
Have you been able to run these applications and see if they are working as intended?  If so, you should delete the commented out lines.